### ISSUE TITLE
is h2o aiming to be the fastest c server in the world?

### ISSUE THREAD

**Issue by superdolt** [h2o/h2o#2837]
is h2o aiming to be the fastest c server in the world?

coz i'm only interested in h2o if it's so, so just curious

**Comment by utrenkner** [comment]
I do not think you will get an answer to such a general question. What do you consider "fast"? Is it the latency for one request, the number of requests served by second, or the serving of meaningful content of real world web pages (see e.g. https://github.com/andydavies/http2-prioritization-issues).

I am not a H2O developer, but my guess is that the latter is what this project aims to optimize. After all, the development is largely sponsored by a CDN.

**Comment by superdolt** [comment]
1. req/s <- this is considered "fast" then. is it supposed to be the "fastest"?

**Comment by utrenkner** [comment]
I am not an expert on web server benchmarks. But I am 99% sure that you can find software which is "faster", if you just look look for maximum req/s of a static file.

**Comment by superdolt** [comment]
so how much slower than the fastest will h2o be?
